hooks/mock/content/events/upcoming-seminar-series-events.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "date": "June 23, 2023",
+    "startDate": "2023-06-23",
+    "endDate": "",
+    "image": "/images/events/downloaded/recXW4BBNFR06ii2a.jpg",
+    "institution": "University of Maryland ",
+    "location": "YouTube",
+    "speaker": "Cheng-Ju Lin ",
```

Comment:
I think that it might be a good idea to use fake names and data.

This goes for all mock files.

Reply:
Thanks! I have change the data for everything except for `ecosystem`... for me it looks ok to be the real data but if you think is better to mock this I can do it